di-ipv-core-stub/compose.yaml

```diff
@@ -0,0 +1,17 @@
+version: "3.9"
```

Comment:
The script & .env file have been swapped for a docker-compose file with environment variables and volumes. The secret stuff is still elsewhere